webui/tests/cypress/integration/02-navigation.spec.ts

```diff
@@ -4,6 +4,9 @@ describe('Navigation', () => {
   const navSelector = '[class*="Navigation_base_"]';
 
   describe('paths', () => {
+    beforeEach(() => {
```

Comment:
for some reason a single before each on the higher level doesn't work :thinking:  I added tests to cypress.spec to see if I can repro but it works fine there.